pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -402,6 +407,9 @@ func (rd *restoreDataProcessor) processRestoreSpanEntry(
 	iter := sst.iter
 	defer sst.cleanup()
 
+	ctx, agg := bulkutil.MakeTracingAggregatorWithSpan(ctx, "SSTBatcher-aggregator", evalCtx.Tracer)
```

Comment:
hmm do we want to create a new aggregator for every restore span entry we process? I don't think this span will be very long-lived and so it will only be aggregating information for each sst we're ingesting and might not even be easy to view in tracez unless we capture a snapshot that catches this span in action.

What do you think about moving this to a higher level maybe outside the `for loop` where `processRestoreSpanEntry` is called in `runRestoreWorkers`. Lets create an aggregator per worker we spawn? This will be longer lived than this span and will aggregate information across SSTs ingested by the worker.

Reply:
done, use an aggregator for each worker instead of each span